diff --git a/apps/www/pages/r/[id].tsx b/apps/www/pages/r/[id].tsx
index 7f582cfa..9d9f9bed 100644
--- a/apps/www/pages/r/[id].tsx
+++ b/apps/www/pages/r/[id].tsx
@@ -16,10 +16,6 @@ interface RoomProps {
 }
 
 export default function Room({ id }: RoomProps) {
-  if (typeof window !== 'undefined' && window.self !== window.top) {
-    return <IFrameWarning url={`https://tldraw.com/r/${id}`} />
-  }
-
   return (
     <>
       <Head>
